app/views/incident_reports/_form.html.haml

```diff
@@ -244,6 +244,32 @@
         prompt: true
   .passenger-incident-info{style: ('display: none;' unless report.passenger_incident?)}
     %h3 Passenger Incident Information
+    .field
+      = form.label :inj_pax_info
+      = form.check_box :inj_pax_info, checked: report.has_injured_passengers?,
+        id: :supervisor_report_inj_pax_info
+    .inj-pax-info{style: ('display: none;' unless report.has_injured_passengers?)}
+      %h3 Injured Passenger Info
+      = form.fields_for :injured_passengers do |pax_form|
+        .pax-fields
+          .field
+            = pax_form.label :name
+            = pax_form.text_field :name
+            = pax_form.label :address
+            = pax_form.text_field :address
+          .field
+            = pax_form.label :nature_of_injury
+            = pax_form.text_field :nature_of_injury
+            = pax_form.label :transported_to_hospital
+            = pax_form.check_box :transported_to_hospital
+          .field
+            = pax_form.label :home_phone
+            = pax_form.text_field :home_phone
+            = pax_form.label :cell_phone
+            = pax_form.text_field :cell_phone
+            = pax_form.label :work_phone
+            = pax_form.text_field :work_phone
+      %button.btn-default.add-pax Add injured passenger info
```

Comment:
Is there a way to delete these entries? I clicked this button a lot and bulked up the page.

Reply:
Opened https://github.com/umts/incidents/issues/133; will resolve outside of this PR.